include/openrave/kinbody.h

```diff
@@ -3561,9 +3561,9 @@ class OPENRAVE_API KinBody : public InterfaceBase
     boost::shared_ptr<rapidjson::Document> _prAssociatedFileEntries; ///< files tag maintaining entries of data files associated with this object
     Transform _baseLinkInBodyTransform; ///< the transform of the base link in the body coordinate frame. The body transform returned is baselink->GetTransform() * _baseLinkInBodyTransform.inverse(). When setting a transform, the base link transform becomes body->GetTransform() * _baseLinkInBodyTransform
     Transform _invBaseLinkInBodyTransform; ///< _baseLinkInBodyTransform.inverse() for speedup
+    mutable std::string __hashKinematicsGeometryDynamics; ///< hash serializing kinematics, dynamics and geometry properties of the KinBody
```

Comment:
@kanbouchou why mutable? doesn't look like you are changing it in a const function

Reply:
Actually, ``OpenRAVE::KinBody::GetKinematicsGeometryHash() const`` could modify this, so I think ``mutable`` is still needed.